Modified for mlt-0.9.0
From 4e96e011aee9b3607791b4c9b904937ab3fe8c00 Mon Sep 17 00:00:00 2001
From: Dan Dennedy <dan@dennedy.org>
Date: Mon, 30 Dec 2013 12:35:49 -0800
Subject: [PATCH] Fix build on Freetype 2.5.

Reported by Patrick Matthhai/Debian.
diff --git a/src/modules/gtk2/producer_pango.c b/src/modules/gtk2/producer_pango.c
index 4969dd2..ba902bd 100644
--- a/src/modules/gtk2/producer_pango.c
+++ b/src/modules/gtk2/producer_pango.c
@@ -25,7 +25,7 @@
 #include <string.h>
 #include <gdk-pixbuf/gdk-pixbuf.h>
 #include <pango/pangoft2.h>
-#include <freetype/freetype.h>
+#include <freetype.h>
 #include <iconv.h>
 #include <pthread.h>
 #include <ctype.h>
-- 
1.8.5.5

39a40
