--- ./src/swig/ruby/build.orig	2016-03-28 05:51:18.787530750 +0100
+++ ./src/swig/ruby/build	2016-03-28 05:58:09.108631370 +0100
@@ -7,6 +7,7 @@ end
 system( "ln -sf ../mlt.i" )
 system( "swig -c++ -ruby -I../../mlt++ -I../.. mlt.i" )
 $CFLAGS += " -I../.. " + (ENV.has_key?('CXXFLAGS')? ENV['CXXFLAGS'] : '')
+$CXXFLAGS += " -I../.. " + (ENV.has_key?('CXXFLAGS')? ENV['CXXFLAGS'] : '')
 $LDFLAGS += " -L../../mlt++ -lmlt++"
 create_makefile('mlt')
 system( "make" )
